tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -63,6 +66,699 @@ public function test_wp_enqueue_script() {
 		$this->assertSame( '', get_echo( 'wp_print_scripts' ) );
 	}
 
+	/**
+	 * Test standalone and non standalone inline scripts in the 'after' position of a single main script.
```

Comment:
```suggestion
	 * Test standalone and non standalone inline scripts in the 'after' position of a single main script.
```
Can we alter the test method descriptions to begin with "Tests that X does Y (when Z).", so that it's clear what is being tested and is expected?

🔢 Applies to other test methods in the PR.